quipucords/scanner/network/inspect_callback.py

```diff
@@ -59,7 +59,7 @@ def task_on_ok(self, event_dict: dict):
         host = event_data.get("host", UNKNOWN_HOST)
         task = event_data.get("task")
         role = event_data.get("role")
-        logger.info("[host=%s] processing role='%s' task='%s'", host, role, task)
+        logger.debug("[host=%s] processing role='%s' task='%s'", host, role, task)
```

Comment:
Switching to debug because this is too noisy? This will skip output in customer logs in the default server configuration, right?

Reply:
mmhmmm. The recent logs I got from the field were the first ones with this new log and I just added noise.